persistent/Database/Persist/Sql/Raw.hs

```diff
@@ -74,12 +90,18 @@ getStmt sql = do
     liftIO $ getStmtConn conn sql
 
 getStmtConn :: SqlBackend -> Text -> IO Statement
-getStmtConn conn sql = do
+getStmtConn = getStmtConn' False
+
+getStmtConn' :: Bool -> SqlBackend -> Text -> IO Statement
+getStmtConn' useCursorIfPossible conn sql = do
```

Comment:
I'm not really a fan of `Bool` parameters when we have some specific intent - it makes it hard to read at the use-sites. Introducing a separate data type that specifies the behavior would be preferable IMO.

Reply:
This has been replaced with a sum type.